src/store/login/saga.ts

```diff
@@ -132,6 +133,9 @@ function* listenForLoginEvents() {
   while (true) {
     yield take(authChannel, AuthEvents.UserLogin);
 
+    // After successful login
+    yield spawn(openFirstConversationAfterChannelsLoaded);
```

Comment:
Don't have to move this now but I'd consider which piece of the app this impacts. Maybe a listener should exist in the area where the activeConversationId is maintained. That would isolate this logic from the login logic more cleanly. If that area of the app ever gets refactored/removed then the login code would never need to be touched.

Again, don't have to change this now. Just trying to point out alternative design choices.